tests/e2e/export.go

```diff
@@ -95,3 +96,13 @@ func exportResourceAsUnstructured(h *create.Harness, obj *unstructured.Unstructu
 	}
 	return u
 }
+
+func getKRMObject(h *create.Harness, obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
```

Comment:
The goal here is to move some logic out of unified_test so that it doesn't become larger and larger, and to improve readability. I believe it is acceptable to put this brief logic back into unified_test. I borrowed the code from [script_test.go](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/script_test.go#L149)

I think we'll need to refactor the unified_test:
1. reorganize and document the usage of env variables(b/332719149)
2. move reusable code to util files